WalletWasabi.Backend/wwwroot/css/bootstrap.css

```diff
@@ -1,7 +1,11650 @@
-/*# sourceMappingURL=bootstrap.min.css.map */
\ No newline at end of file
+ */
+
```

Comment:
@CAnorbo Can you verify these changes?

Reply:
But wasn't that the result of minifying?

Reply:
I think it was minified so this change is wrong and decrease the performance of the website.

Reply:
@CAnorbo does this make sense? If no then we can merge.

Reply:
True, it's better if minified, but not a big problem.